Example/Database/Tests/Helpers/FEventTester.m

```diff
@@ -132,7 +132,7 @@ - (void) waitForInitialization {
         NSRange theRange;
         theRange.location = 0;
         theRange.length = self.initializationEvents;
-        [actualPathsAndEvents removeObjectsInRange:theRange];
+        [self->actualPathsAndEvents removeObjectsInRange:theRange];
```

Comment:
I may be missing something, but (without me digging to deeply) can this be `self.actualPathsAndEvents`?